protos/cryptopb/cryptopb.proto

```diff
@@ -39,7 +39,6 @@ message SignRequest {
 
 message SignResult {
   option (mir.event) = true;
-
```

Comment:
I find it a bit more readable to have an empty line after these annotations. I've tried to consistently use it across all `.proto` files.

Reply:
done, the line being deleted was an unintended artifact.